first_hour_on_vwb/working_with_groups.ipynb

```diff
@@ -156,17 +248,28 @@
     "## Create a new Verily Workbench group\n",
     "<a id='create_group'></a>\n",
     "\n",
-    "Run the cell below to create a widget, then populate the widget's fields and click the button to create a new Verily Workbench group. \n",
-    "The value provided for the `GROUP_NAME` should be unique, and reflect the purpose of the group (for example, for a group of Verily researchers working on the 1000 Genomes dataset, `verily-1000-genomes-researchers` would be appropriate).\n",
+    "Run the cell below to create a widget that creates an org-linked Workbench group. Then, populate the widget's input fields:\n",
     "\n",
+    "- `Group Name`: Should be unique and reflect the purpose of the group (for example, for a group of Verily researchers working on the 1000 Genomes dataset, `verily-1000-genomes-researchers` would be appropriate).\n",
+    "- `Org ID`: Your organization's ID.\n",
+    "- `Expiration`: The default and maximimum MEMBER role expiration in *days*. Zero means the MEMBER role does not expire.\n",
+    "- `Expiration Notification`: Notify group admins about member expirations.\n",
+    "- `Require Grant Reason`: Require a reason on grants to the group.\n",
+    "\n",
+    "Click the button to create a new Verily Workbench group. \n",
     "You should see output resembling:\n",
     "\n",
     "```\n",
     "Workbench group created.\n",
     "<GROUP_NAME>\n",
-    "  Email: <GROUP_NAME>@verily.com\n",
-    "  # Members: 1\n",
-    "  Current user's policies: ADMIN\n",
+    "  Organization           : <ORG_ID>\n",
```

Comment:
I admit I don't have a great mental model for what's happening here, but if those commented HTML strings will affect what should be written here, this is a reminder to update here accordingly